assets/src/adaptivity/capi.ts

```diff
@@ -98,12 +99,70 @@ export interface ICapiVariableOptions {
   writeonly?: boolean;
   allowedValues?: string[];
   bindTo?: string;
+  shouldConvertNumbers?: boolean;
 }
 
 export interface JanusConditionProperties extends ConditionProperties {
   id: string;
   type?: CapiVariableTypes;
 }
+export const parseArrayWithStringConversion = (val: unknown): unknown[] => {
```

Comment:
isn't this *without* string conversion? this version is *not* converting the numbers to numbers that are strings